src/components/ProductList.tsx

```diff
@@ -26,43 +29,57 @@ const ProductList = ({ actionLabel, content, category }: ProductListProps) => {
 
   return (
     <div className="w-full">
-      <h3
-        id={CATEGORY_NAME}
-        className="mb-0 mt-10 border-b-2 border-border pb-4 text-2xl"
-      >
-        {category}
-      </h3>
-      <List aria-labelledby={CATEGORY_NAME} className="m-0">
-        {content.map(({ title, description, link, image, alt, id }, idx) => (
-          <ListItem key={id || idx} color="text" className="mb-0 mt-8 flex">
-            <div className="w-20">
-              {image && (
-                <Image
-                  src={image}
-                  alt={alt}
-                  width={66}
-                  className="rounded-sm shadow-lg dark:shadow-body-light"
-                />
+      {category && (
+        <h3
+          id={CATEGORY_NAME}
+          className="mb-0 mt-10 border-b-2 border-border pb-4 text-2xl"
+        >
+          {category}
+        </h3>
+      )}
+      <List aria-labelledby={CATEGORY_NAME} className="m-0 mb-4">
+        {content.map(
+          ({ title, description, link, image, alt, id, contentItems }, idx) => (
+            <ListItem
+              key={id || idx}
+              color="text"
+              className={cn(
+                "mb-0 mt-8 flex pb-4",
+                idx !== content.length - 1 && "border-b"
               )}
-            </div>
-            <Flex className="ms-4 w-full flex-col justify-between border-b pb-4 sm:ms-6 sm:flex-row">
-              <div className="flex-1">
-                <div>{title}</div>
-                <div className="mb-0 text-sm opacity-60">{description}</div>
+            >
+              <div className="w-20">
+                {image && (
+                  <Image
+                    src={image}
+                    alt={alt}
+                    width={66}
+                    className="rounded-xl shadow-lg dark:shadow-body-light"
+                  />
+                )}
               </div>
-              {link && (
-                <ButtonLink
-                  variant="outline"
-                  href={link}
-                  className="ms-0 mt-4 min-h-fit gap-0 self-center rounded-sm px-6 py-1 sm:ms-8 sm:mt-0"
-                >
-                  {actionLabel}
-                  <VisuallyHidden>to {title} website</VisuallyHidden>
-                </ButtonLink>
-              )}
-            </Flex>
-          </ListItem>
-        ))}
+              <Flex className="ms-4 w-full flex-col justify-between pb-4 sm:flex-row">
+                <div className="flex flex-1 flex-col gap-2">
+                  <div>{title}</div>
+                  <div className="mb-0 text-sm opacity-60">{description}</div>
+                  <div className="mb-0 flex flex-col gap-2 text-sm opacity-60">
+                    {contentItems}
```

Comment:
Given that `contentItems` is optional, shouldn't we consider that case and avoid rendering the wrapping div?